doc/funding/process.md

```diff
@@ -1,10 +1,15 @@
 # Process for funding
 
-Our process for funding is still a working draft.
 This is the process the Board currently follows, and it will improve the details of this process as we begin taking our first steps.
 
 - The [Board Chair](#board-chair) is accountable for ensuring the Board understands its budget.
 - [Subcommittees](#subcommittees) (or individual board members) make proposals for funding to the Board.
 - The Board takes a vote on whether to fund each proposal.
 - If proposals are accepted, the Board will delegate the implementation of the proposal to a subcommittee or individual on the Board.
 - If subcommittees are responsible for implementing a proposal, then a **majority vote of the Governing Board members on the subcommittee** makes decision about how to spend the funds, as long as it is under the total amount in the proposal.   
+
+## How to submit a proposal for funding
+
+- Go to [this proposal template](https://docs.google.com/document/d/101cNUZbGCWUMn8LbDiSRjL5Q78e_31gN4-FqCiSp47o/edit?usp=sharing)
```

Comment:
Should this template be public? Right now the permissions only allow governing board members to open it, I think.

Reply:
I'll plan to make it public!

Reply:
@choldgraf I think we should add some detail to this process around ensuring some of the budgeting math is explained in some level of detail for transparency. Ex. What is the industry standard hourly cost for a service of this type and how was that used to formulate a specific line-item in the budget proposal?